### PR TITLE
[IC-134] New env variable for messages report

### PR DESCRIPTION
**Pull request by gquadrati**
### List of changes

- add connection strings to `io-p-cosmos-api-northeurope`
- add connection string to `iopstapireplica`
- add connection string to `iopstcdnassets` for reading `visible-services-extended.json`
- add new blob containers `messages-report-step1` and `messages-report-step-final`
- add new variable for new handlers, used for tuning cosmos and blob storage queries

### Motivation and context

In order to retrieve aggregated data for messages, ad new handler triggered by a command from EH topic that will scan messages and write a csv with required aggregated data.

NOTE: this is just a workaroud to get data quickly.

### Type of changes

- [x] Add new resources
- [x] Update configuration to existing resources
- [ ] Remove existing resources

### Env to apply

- [ ] DEV
- [ ] UAT
- [x] PROD

### Does this introduce a change to production resources with possible user impact?

- [ ] Yes, users may be impacted applying this change
- [ ] No

### Does this introduce an unwanted change on infrastructure? Check terraform plan execution result

- [ ] Yes
- [ ] No

### Other information

---

### CODE REVIEW

File: src/core/function_elt.tf [hunk 1]
```diff
@@ -78,6 +93,20 @@ module "function_elt" {
 
     PROFILE_TOPIC_NAME              = "io-cosmosdb-profiles"
     PROFILE_TOPIC_CONNECTION_STRING = module.event_hub.keys["io-cosmosdb-profiles.io-fn-elt"].primary_connection_string
+
+    COSMOSDB_REPLICA_NAME = "db"
+    COSMOSDB_REPLICA_URI  = replace(data.azurerm_cosmosdb_account.cosmos_api.endpoint, "io-p-cosmos-api", "io-p-cosmos-api-northeurope")
```
Comment (pasqualedevita) [inline src/core/function_elt.tf:98]:
```suggestion
    COSMOSDB_REPLICA_URI  = "${data.azurerm_cosmosdb_account.cosmos_api.endpoint}-northeurope"
```

**Review by pasqualedevita — COMMENTED** [review]
only for style

**Review by pasqualedevita — APPROVED** [review]

File: src/core/function_elt.tf (hunk 1 above)
Comment (gquadrati) [reply src/core/function_elt.tf:98]: Not sure it's gonna work this way
I'm replacing those strings into an url like `https://<name>.documents.azure.com:443/`

Comment (pasqualedevita) [reply src/core/function_elt.tf:98]: ok u're right

**Comment by pasqualedevita** [comment]
we need to apply this pr before https://github.com/pagopa/io-infra/pull/53

**Comment by gquadrati** [comment]
I'm gonna wait the approval of https://github.com/pagopa/io-functions-elt/pull/21 before apply it.

**Pull request merged**